server/validators/dateIsAfter.ts

```diff
@@ -1,10 +1,10 @@
 import { registerDecorator, ValidationOptions } from 'class-validator'
-import { isValid, startOfDay } from 'date-fns'
+import { isValid } from 'date-fns'
 import SimpleDate from '../commonValidationTypes/simpleDate'
 
 export default function DateIsAfter(dateToCompare: Date, validationOptions?: ValidationOptions) {
   const dateIsAfter = (date: SimpleDate) =>
-    date !== undefined && isValid(date.toRichDate()) ? startOfDay(date.toRichDate()) > startOfDay(dateToCompare) : true
+    date !== undefined && isValid(date.toRichDate()) ? date.toIsoString() > dateToCompare.toISOString() : true
 
```

Comment:
Are there any tests to tickle the issue with the original lines? And that subsequently pass with the new lines?